### PR TITLE
Update read me instructions for running data hub frontend natively

### PR DESCRIPTION
**Pull request by chloehkwong1**
## Description of change

This PR updates and clarifies the read me instructions on how to get Data Hub running natively. Included are instructions on how to use different APIs. 

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"

- [ ] Has the branch been rebased to master?
- [ ] Automated tests (Any of the following when applicable: Unit, Functional or End-to-End)
- [ ] Manual compatibility testing (Browsers: Chrome, Firefox, Edge, Safari)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,39 +60,71 @@ Please view the dedicated [Docker readme](./docs/Docker.md).
...
-    cd test/sandbox
-    docker build -t data-hub-sandbox .
-    docker run --rm --name data-hub-sandbox -it -p 8001:8000 data-hub-sandbox
+    docker run -it -p 6379:6379 redis:3.2
     ```
 
-8.  Start the node server
+9. Go to the Django admin site for your relevant environment and get an access token by adding the following path `/add-access-token/` to the URL. Add a frontend environment variable `OAUTH2_DEV_TOKEN` and set this as equal to the access token. 
+
+10.  Start the node server
```
Comment (mforner13) [inline README.md:127]: The formatting is a bit off for this section. It's got the whole instructions in a code block I think

Screenshot 2022-01-10 at 15 00 05

File: README.md [hunk 2]
```diff
@@ -114,9 +146,26 @@ Please view the dedicated [Docker readme](./docs/Docker.md).
...
+
+To use the mock SSO service, add the environment variables from `sample.env` below the line 'To use mock-SSO' into your `.env` file and uncomment.
+
+Set `SSO_ENABLED=true`
+
+  Start the mocked SSO:
+
+    ```bash
+    docker run -it -p 8080:8080 gcr.io/sre-docker-registry/github.com/uktrade/mock-sso:latest
+    ```
```
Comment (mforner13) [inline README.md:161]: Formatting off here too

Screenshot 2022-01-10 at 15 01 32

**Review by mforner13 — COMMENTED** [review]
A few suggestions :)

File: README.md [hunk 3]
```diff
@@ -60,39 +60,71 @@ Please view the dedicated [Docker readme](./docs/Docker.md).
...
+    ``` 
+    DATA_HUB_BACKEND_ACCESS_KEY_ID=frontend-key-id
+    DATA_HUB_BACKEND_SECRET_ACCESS_KEY=frontend-key
+    ```
+
+7. The environment variables copied from Vault and `sample.env` are set up for running both the frontend and the API using the docker. To run the frontend natively, the following variables will need to be changed to:
+    ```
+    REDIS_HOST=localhost
+    REDIS_URL=redis://localhost:6379
+    ```
```
[the remark this answers is not in the document]
Comment (chloehkwong1) [reply README.md:111]: Thanks for going through this @mforner13 !  I'm not sure what you mean by this comment, please could you elaborate? No. 5 is looking at Playbook, No.6 is Vault, and No.7 just changing the redis URLs. Thanks :D

[the remark this answers is not in the document]
Comment (mforner13) [reply README.md]: Ah sorry I was looking at number 5 in the other section and thought it was above (ie applying to both scenarios) - but all good! :)

File: README.md [hunk 4]
```diff
@@ -60,39 +60,71 @@ Please view the dedicated [Docker readme](./docs/Docker.md).
...
-    docker run -it -p 6379:6379 redis:3.2
+    npm run develop
     ```
 
-6.  Start the mocked SSO:
+    The server will watch for changes and rebuild sass or compile js using webpack as
+    needed. Changes to server side code will result in the server autorestarting.
+    The server will run with the node debug flag so you can debug with Webstorm
+    or Visual Studio Code.
+
```
[the remark this answers is not in the document]
Comment (mforner13) [reply README.md:104]: Ah yes I must have been thinking about how you can link to sections but seems like what you've got now is a good solution :)

File: README.md [hunk 5]
```diff
@@ -60,47 +60,86 @@ Please view the dedicated [Docker readme](./docs/Docker.md).
...
+5.  Go to the [Playbook](https://readme.trade.gov.uk/docs/playbooks/datahub.html#environments) and use the Admin URLs to set the environment variable `API_ROOT`. You will need to remove `/admin` from the end of the API paths.
+
+6.  Go to Vault, look in datahub-fe, and click on the environment you want to use. Change the following environment variables in your `.env` file to the ones specified in Vault:
+    ``` 
+    DATA_HUB_BACKEND_ACCESS_KEY_ID=frontend-key-id
+    DATA_HUB_BACKEND_SECRET_ACCESS_KEY=frontend-key
+    ```
+
+7.  The environment variables copied from Vault and `sample.env` are set up for running both the frontend and the API using the docker. To run the frontend natively, the following variables will need to be changed to:
```
Comment (mforner13) [inline README.md:115]: I think the ones you copy from vault wouldn't be set up for running in docker as they're for the live environments, so think you can just say 'copies from `sample.env`.

**Review by mforner13 — COMMENTED** [review]
Just one last suggestion :)

File: README.md (hunk 5 above)
Comment (chloehkwong1) [reply README.md:115]: Good spot thanks Miriam!! I don't know how that slipped in there arghh!!

**Review by mforner13 — APPROVED** [review]
Looks good! Future new developers thank you 🙏

**Review by cgsunkel — APPROVED** [review]

**Review by tomkingston — APPROVED** [review]
Looking good 👍

**Pull request merged**